test/acceptance/index.js

```diff
@@ -79,6 +79,18 @@ describe('index', function () {
     });
   });
 
+  describe('shims', function() {
+    it('should return the expected result', function (done) {
+      var opts = { shim: true, config: 'tmp/shim-config.js', exclude: ['underscore'] };
+      require('../../lib')(opts, function () {
```

Comment:
These tests could use some bigger refactor here it seems but I was just keeping with patterns of existing tests.